src/Utils.js

```diff
@@ -116,7 +116,26 @@ export function selectorError(selector) {
   );
 }
 
-export const isCompoundSelector = /[a-z]\.[a-z]/i;
+export const isCompoundSelector = /([a-z]\.[a-z]|[a-z]\[.*\])/i;
```

Comment:
My testing shows that it doesnt?  `div[foo][bar]` matches but `[foo][bar]` does not